docs/development/TESTS.md

```
@@ -72,6 +72,15 @@ uv run pytest tests/property/ -v --hypothesis-max-examples=1000
 uv run pytest tests/property/ -v --hypothesis-verbosity=verbose
 ```
 
+### Opting in to live DEBUG logs
+
+By default, the template disables live pytest CLI logging (`log_cli = false`) to keep normal test output concise.
+When you need detailed live logs for debugging, enable them per-run:
+
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code fence style
Expected: tilde; Actual: backtick

(MD048, code-fence-style)